[Bug] macOS builds hang pretty often

# 🐛 Bug report

## Description

macOS build on Azure Pipelines have a tendency to hang and timeout. Looks like this environment is cursed to us because right now it is not that Chrome isn't getting installed, it's just that test hang.

```
* packages/hint-content-type

$ nyc ava

  ✔ [jsdom] HTML page is served without 'Content-Type' header (7.9s)
  ✖ dist/tests/tests.js exited due to SIGSEGV

  1 test passed

##[error]The operation was canceled.
##[section]Finishing: Test
```

Not sure if it is related to Chrome or something else entirely (I think it's probably Chrome). Need to find out what's going on.

--- a/packages/hint-content-type/tests/tests.ts
+++ b/packages/hint-content-type/tests/tests.ts
@@ -10,7 +10,7 @@ const { generateHTMLPage, getHintPath } = test;
 const hintPath = getHintPath(__filename);
 
 const pngImage = fs.readFileSync(`${__dirname}/fixtures/image.png`); // eslint-disable-line no-sync
-const svgImage = '<svg xmlns="http://www.w3.org/2000/svg"><path d="M1,1"/></svg>';
+const svgImage = '<svg xmlns="http://www.w3.org/2000/svg" width="1px" height="1px"><path d="M1,1"/></svg>';
 
 const incorrectCharsetErrorMessage = `'content-type' header charset value should be 'utf-8', not 'iso-8859-1'.`;
 const invalidMediaTypeErrorMessage = `'content-type' header value should be valid (invalid media type).`;

